test/parallel/test-tls-wrap-econnreset.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const https = require('https');
+const net = require('net');
+
+net.createServer(c => {
+  setTimeout(() => c.end(), 300);
+}).listen(common.PORT, () => {
+  https.get(`https://localhost:${common.PORT}`, (res) => {
+    throw new Error('this request must return error');
```

Comment:
I think we are moving from `common.PORT` to use _random_ ports by using `listen(0)` and then `server.address().port`.